content/applications/websites/livechat.rst

```diff
@@ -242,46 +265,63 @@ If a users :guilabel:`Online Chat Name` is not set, the name displayed will defa
+
+.. note::
+   Conversations are assigned to operators based on a number of criteria, including availability and
+   the number of ongoing conversations. While the operator's main language and additional languages
+   are taken into consideration, they do **not** supersede all other criteria.
+
 Join or leave a channel
 -----------------------
 
-To join a live chat channel, go to the :menuselection:`Live Chat` app and click the :guilabel:`JOIN`
-button on the kanban card for the appropriate channel.
+To join a live chat channel, go to the :menuselection:`Live Chat` app and click the :guilabel:`Join`
+button on the Kanban card for the appropriate channel.
 
-Any channel where the user is currently active will show a :guilabel:`LEAVE` button. Click this
-button to disconnect from the channel.
+Any channel where the user is currently active shows a :guilabel:`Leave` button. Click this button
+to disconnect from the channel.
 
 .. image:: livechat/leave-channel.png
    :align: center
    :alt: View of a channel form and the option to join a channel for Odoo Live Chat.
 
 .. important::
-   *Operators* that do not show any activity in Odoo for more than thirty minutes will be considered
+   *Operators* that do not show any activity in Odoo for more than thirty minutes are considered
    disconnected, and subsequently removed from the channel.
 
 Manage live chat requests
 -------------------------
 
-When an operator is active in a channel, chat windows will open in the bottom right corner of the
+When an operator is active in a channel, chat windows open in the bottom-right corner of the
 screen, no matter where they are in the database. They can participate in conversations without
 leaving their current location.
 
 .. tip::
-   Conversations can also be accessed by clicking the :guilabel:`Conversations` icon in the
-   menu bar.
+   Conversations can also be accessed by clicking the :guilabel:`Conversations` icon in the menu
+   bar.
 
    .. image:: livechat/menu-bar.png
       :align: center
       :alt: View of the menu bar in Odoo emphasizing the conversations icon.
 
-Live chat conversations can also be viewed by navigating to :menuselection:`Dashboard --> Discuss`.
-New conversations will appear in bold under the :guilabel:`LIVECHAT` heading along the left panel.
+Live chat conversations can also be viewed by navigating to :menuselection:`Dashboard -->
+Discuss app`. New conversations appear in bold under the :guilabel:`LIVECHAT` heading along the left
+panel.
 
 .. image:: livechat/managing-chat-responses.png
    :align: center
    :alt: View of the discuss application with a message sent through live chat in Odoo.
 
-Click on a conversation in the left panel to select it. This will open the conversation. From this
+Click on a conversation in the left panel to select it. This opens the conversation. From this
 view, an operator can participate in the chat the same as they would in the normal chat window.
 
 .. seealso::
```

Comment:
This looks a little wonky: maybe put in a block or add a title? 

![2024-03-21_15-36](https://github.com/odoo/documentation/assets/91896312/a80cf301-84d3-45f7-8665-e0319287f52a)